Fix failing unit test in Statusboard

## Describe the bug

There is a failing unit test in the statusboard's test suite.

## To Reproduce

Steps to reproduce the behavior:
1. run the test suite in `statusboard/package.json`
2. press "a" to run all
3. Get `Test Suites: 1 failed, 4 passed, 5 total` due to `Cannot find module './Projects' from 'Projects.test.js'`

## Expected behavior

There should never be failing unit tests in the `main` branch.

--- a/statusboard/src/pages/Projects.test.js
+++ b/statusboard/src/pages/Projects.test.js
@@ -4,7 +4,7 @@ import React from 'react';
 
 import { render, screen } from '@testing-library/react';
 
-import Projects from './Projects';
+import Projects from './Projects/Projects';
 import { SAMPLE_BRIGADE } from '../utils/utils.test';
 
 const fakeServer = setupServer(

--- a/statusboard/src/pages/Projects.test.js
+++ b/statusboard/src/pages/Projects.test.js
@@ -1,11 +1,13 @@
 import { rest } from 'msw';
 import { setupServer } from 'msw/node';
 import React from 'react';
+import { StaticRouter } from 'react-router-dom';
 
 import { render, screen } from '@testing-library/react';
 
 import Projects from './Projects/Projects';
 import { SAMPLE_BRIGADE } from '../utils/utils.test';
+import { BrigadeDataContextProvider } from '../contexts/BrigadeDataContext';
 
 const fakeServer = setupServer(
   rest.get('/api/data.json', (req, res, ctx) => {
@@ -20,8 +22,14 @@ afterEach(() => fakeServer.resetHandlers());
 
 describe('Page: <Projects>', () => {
   it('renders before and after projects have loaded', async () => {
-    render(<Projects />);
-    expect(await screen.findByText(/Loading/i)).toBeInTheDocument();
+    render(
+      <BrigadeDataContextProvider>
+        <StaticRouter location="/projects?timeRange=all%20time">
+          <Projects />
+        </StaticRouter>
+      </BrigadeDataContextProvider>,
+    );
+    expect(await screen.getByText(/Loading/i)).toBeInTheDocument();
     expect(await screen.findByText(/311-index/)).toBeInTheDocument();
   });
 });

--- a/statusboard/src/pages/Projects.test.js
+++ b/statusboard/src/pages/Projects.test.js
@@ -12,7 +12,7 @@ import { BrigadeDataContextProvider } from '../contexts/BrigadeDataContext';
 const fakeServer = setupServer(
   rest.get('/api/data.json', (req, res, ctx) => {
     return res(ctx.delay(), ctx.json([SAMPLE_BRIGADE]));
-  })
+  }),
 );
 
 // from: https://testing-library.com/docs/react-testing-library/example-intro
@@ -20,14 +20,20 @@ beforeAll(() => fakeServer.listen());
 afterAll(() => fakeServer.close());
 afterEach(() => fakeServer.resetHandlers());
 
+function RouteWithContext({ children, location }) {
+  return (
+    <BrigadeDataContextProvider>
+      <StaticRouter location={location}>{children}</StaticRouter>
+    </BrigadeDataContextProvider>
+  );
+}
+
 describe('Page: <Projects>', () => {
   it('renders before and after projects have loaded', async () => {
     render(
-      <BrigadeDataContextProvider>
-        <StaticRouter location="/projects?timeRange=all%20time">
-          <Projects />
-        </StaticRouter>
-      </BrigadeDataContextProvider>,
+      <RouteWithContext location="/projects?timeRange=all%20time">
+        <Projects />
+      </RouteWithContext>,
     );
     expect(await screen.getByText(/Loading/i)).toBeInTheDocument();
     expect(await screen.findByText(/311-index/)).toBeInTheDocument();

--- a/statusboard/src/pages/Projects.test.js
+++ b/statusboard/src/pages/Projects.test.js
@@ -12,7 +12,7 @@ import { BrigadeDataContextProvider } from '../contexts/BrigadeDataContext';
 const fakeServer = setupServer(
   rest.get('/api/data.json', (req, res, ctx) => {
     return res(ctx.delay(), ctx.json([SAMPLE_BRIGADE]));
-  }),
+  })
 );
 
 // from: https://testing-library.com/docs/react-testing-library/example-intro
@@ -33,7 +33,7 @@ describe('Page: <Projects>', () => {
     render(
       <RouteWithContext location="/projects?timeRange=all%20time">
         <Projects />
-      </RouteWithContext>,
+      </RouteWithContext>
     );
     expect(await screen.getByText(/Loading/i)).toBeInTheDocument();
     expect(await screen.findByText(/311-index/)).toBeInTheDocument();

--- a/statusboard/src/utils/utils.ts
+++ b/statusboard/src/utils/utils.ts
@@ -104,9 +104,7 @@ export function filterProjectsByTime(
 ) {
   const timeRanges = timeRangeKey ? ACTIVE_THRESHOLDS[timeRangeKey] : undefined;
   if (!timeRanges) return projects;
-  return projects.filter((p) =>
-    p.last_pushed_within ? timeRanges.includes(p.last_pushed_within) : false
-  );
+  return projects.filter((p) => timeRanges.includes(p.last_pushed_within));
 }
 
 export function filterActiveProjects(

